features/promotion/managing_promotions/seeing_list_coupons_button_if_promotion_has_coupon_based.feature

```diff
@@ -0,0 +1,24 @@
+@managing_promotions
+Feature: Seeing list coupons button if promotion has coupon based
+    In order to facilitate work with the management of coupons
+    As an Administrator
+    I want to be prevented from manage coupons without specifying required fields
+
+    Background:
+        Given the store operates on a single channel in "United States"
+        And there is a promotion "Christmas sale"
+        And it is coupon based promotion
+        And I am logged in as an administrator
+
+    @ui
+    Scenario: Edit promotion and seeing the button "Manage coupons"
```

Comment:
```suggestion
    Scenario: Editing a promotion and seeing the "Manage coupons" button
```

Reply:
Maybe `Being able to manage promotion's coupons`? `Seeing the button` is too technical